Resources/views/Configuration/config.js.twig

```diff
@@ -44,6 +44,8 @@
                 .icon('<span class="glyphicon glyphicon-{{ entity.icon }}"></span>');
 
             {{ entity.name }}.dashboardView()
+                .title('Recent {{ entity.name }}')
```

Comment:
Any time estimation before the 0.7.0 release?

Changing it to plural would be nice, while your 0.7.0 is not merged. `Recent post` with 5 rows looks weird to me. :)